sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcColumnarBatchReaderSuite.scala

```diff
@@ -77,4 +77,43 @@ class OrcColumnarBatchReaderSuite extends QueryTest with SharedSparkSession {
       assert(p1.getUTF8String(0) === partitionValues.getUTF8String(0))
     }
   }
+
+  test("orc data created by the hive tables having _col fields name") {
```

Comment:
Plz add the prefix `test("SPARK-32234: orc data...`.

Reply:
done